bin/hpcbind

```diff
@@ -635,7 +635,9 @@ if [[ ${HPCBIND_TEE} -eq 0 || ${HPCBIND_VERBOSE} -eq 0 ]]; then
 
   if [[ ${HPCBIND_HAS_HWLOC} -eq 1 ]]; then
     echo "[BINDINGS]" >> ${HPCBIND_LOG}
-    hwloc-ls --restrict "${HPCBIND_HWLOC_CPUSET}" >> ${HPCBIND_LOG}
+    if [[ ${HPCBIND_LSTOPO} -eq 1 && ! -z ${DISPLAY} ]]; then
+      hwloc-ls --restrict "${HPCBIND_HWLOC_CPUSET}" >> ${HPCBIND_LOG}
```

Comment:
Isn't `$DISPLAY` set / unset on the user's machine?  Assuming that's the case, then conditioning `hwloc-ls --restrict  "${HPCBIND_HWLOC_CPUSET}" >> ${HPCBIND_LOG}` upon `${DISPLAY}` being non zero (if it's set at all) could have negative downstream effects for users who wish to use `hpcbind` , but do not have `$DISPLAY` set.  I checked a couple of (non-Windows) machines, and `$DISPLAY` wasn't set, appearing to be the default scenario.  Would it be possible for you to simply `unset $DISPLAY` before using `hpcbind`?